addons/l10n_in_edi_ewaybill/wizard/l10n_in_ewaybill_cancel.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class EwaybillCancel(models.TransientModel):
+
+    _name = 'l10n.in.ewaybill.cancel'
+    _description = 'Cancel Ewaybill'
+
+    move_id = fields.Many2one('account.move')
```

Comment:
not in stable

Reply:
I see we already have a request edi cancellation button (at least there is a cancel possibility in _get_move_applicability which probably works in a confusing way as we will mostly have multiple EDIs on the same invoice, so I wonder if it is really important here.  )

Reply:
Hello @jco-odoo, 

I've removed the logic for transferring the cancellation of the e-way bill. 
This functionality move to another task. 
Can I continue with improving other views using the stable version?